.github/pytorch-probot.yml

```diff
@@ -4,3 +4,5 @@ ciflow_push_tags:
 - ciflow/trunk
 - ciflow/binaries
 - ciflow/binaries/all
+- ciflow/periodic
+- ciflow/periodic/all
```

Comment:
Nit: you don't need `ciflow/periodic/all`, just `ciflow/periodic` is enough.  `ciflow/binaries/all` is just an outlier, not the norm